hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreation.java

```diff
@@ -1313,6 +1313,10 @@ public void testFileIdMismatch() throws IOException {
         fail();
       } catch(FileNotFoundException e) {
         FileSystem.LOG.info("Caught Expected FileNotFoundException: ", e);
+      } catch (AssertionError ae) {
```

Comment:
Handling AssertionError is not a good idea. But, this is -ve test case, if we dont handle Error here then we need to remove the assertion statement in code. 
If any alternative approach, we shall discuss and proceed.